tmt/cli.py

```diff
@@ -546,8 +546,11 @@ def import_(
 @click.pass_context
 @name_filter_condition_long
 @click.option(
-    '--nitrate', is_flag=True,
-    help='Export test metadata to Nitrate.')
+    '-h', '--how', metavar='METHOD',
```

Comment:
this would just be moving the real usage of --how into 2.0, everyone would just keep using --nitrate...well, maybe newcomers would use --how

but I'll let @psss decide this one

Reply:
Yeah, makes sense to me as well to keep `--nitrate` as a hiddne option with warning at least until `tmt-2.0`. There's already a bunch of scripts written and being used for mass migration, would be better not to break them.